OpenSim/Common/TableSource.h

```diff
@@ -202,21 +202,30 @@ class TableSource_ : public Component {
                          TimeOutOfRange, 
                          time, timeCol.front(), timeCol.back());
 
-        const auto colInd = _table.getColumnIndex(columnLabel);
+        const auto colInd = 
+            static_cast<int>(_table.getColumnIndex(columnLabel));
```

Comment:
I believe unsigned or size_t is the correct return type. It represents index correctly -- as a non-negative integer. I have used this in my code.

I am not sure why Storage was written differently. I think there are a lot of places where an int is (incorrectly) used as an argument or return type in OpenSim.